src/js/components/locations-configuration/AddZoneModal.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+
+import PropTypes from 'prop-types';
+import { getTranslate } from 'react-localize-redux';
+import { connect } from 'react-redux';
+import Alert from 'react-s-alert';
+
+import { hideSpinner, showSpinner } from 'actions';
+import ModalWrapper from 'components/form-elements/ModalWrapper';
+import apiClient from 'utils/apiClient';
+import Translate, { translateWithDefaultMessage } from 'utils/Translate';
+
+
+class AddZoneModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      values: {
+        active: true,
+      },
+    };
+  }
+
+  handleSubmit(values) {
+    this.props.showSpinner();
+    apiClient.post('/openboxes/api/locations/', {
+      ...values,
+      'parentLocation.id': `${this.props.locationId}`,
+      'locationType.id': 'ZONE',
```

Comment:
It should not be added like that. We do not have certainty what id this type will have in some other databases. You can take the id for this from the Location Type options, where locationTypeCode is 'ZONE".

Reply:
(same will be for Bin Location in your next task)